teachertool/src/utils/browserUtils.ts

```diff
@@ -0,0 +1,12 @@
+export const isLocal = () => {
+    return window.location.hostname === "localhost";
+}
+
+export const getEditorUrl = (embedUrl: string) => {
```

Comment:
I think a comment explaining what this returns (maybe with an example) would be super helpful.